.github/team.json

```diff
@@ -3,6 +3,7 @@
     {
       "name": "Querying Components",
       "label": ".Team/QueryingComponents",
+      "projectUrl": "https://github.com/orgs/metabase/projects/50/views/1",
```

Comment:
```suggestion
      "projectUrl": "https://github.com/orgs/metabase/projects/74",
```

Reply:
Good catch, thanks. I wanted to add this url to the QP team. And @NevRA just told me they don't need PR auto-assignment